packages/office-ui-fabric-react/src/components/OverflowSet/examples/OverflowSet.Basic.Example.tsx

```diff
@@ -0,0 +1,136 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import { BaseComponent, css } from 'office-ui-fabric-react/lib/Utilities';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
+import {
+  OverflowSet
+} from 'office-ui-fabric-react/lib/OverflowSet';
+
+import * as stylesImport from './OverflowSet.Example.scss';
+const styles: any = stylesImport;
+
+export class OverflowSetBasicExample extends BaseComponent<any, any> {
+
+  public render() {
```

Comment:
Maybe you should keep the basic example basic and have a "customized" example to customized. Reading the docs this is basically a for loop on items to render them as a given thing, with a menuProps.